### DISCUSSION TITLE
Sorry, could not find Andromeda Galaxy in Stellarium. Please try manually in Stellarium.

### DISCUSSION THREAD

**Discussion by djpierce54** [Stellarium/stellarium#2138] [Q&A]
I have setup the server and tested it in my browser. I can display images etc. Then I wanted to integrate Telescopius in the mold. Setup with the localhost stuff. No matter what object I pick in Telescopius I get the error in the subject line. Although the name of the object changes accordingly. Not sure where my problem is. I tried from the Telescopius website and downloaded and ran frm my Windows 10 desktop. The same.

**Comment by gzotti** [comment]
It seems you must allow CORS globally, i.e. with a star. Do this at your own risk.
EDIT: No, but specify allowed domain with protocol, i.e., `https://telescopius.com`

**Comment by djpierce54** [comment]
Yes I did that. No difference.


Don

On Wednesday, December 29, 2021, 2:09 AM, Georg Zotti ***@***.***> wrote:




It seems you must allow CORS globally, i.e. with a star. Do this at your own risk.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by gzotti** [reply]
I can only say it works for me. Did you relaunch the RemoteControl server as indicated?

**Comment by djpierce54** [comment]
Hmm. I guess I need to work on my network firewall I guess.


Don

On Wednesday, December 29, 2021, 8:29 AM, Georg Zotti ***@***.***> wrote:




I can only say it works for me. Did you relaunch the RemoteControl server as indicated?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
